mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -597,12 +768,13 @@
         <SubgroupPriority value="93"/>
         <WeaponArray Link="HighTemplarWeapon"/>
         <AbilArray Link="attack"/>
+        <Speed value="2.0156"/>
```

Comment:
High Templar's move speed increased from 2.63 to 2.82 (from 1.875 to 2.0156 on normal speed)

Move speed is tiles per second. The patch notes translated it to real seconds on the multiplayer game speed (x1.4)